MeshLib/Elements/Element.h

```diff
@@ -84,7 +84,7 @@ class Element
 	virtual unsigned getDimension() const = 0;
 
 	/// Returns the i-th edge of the element.
-	const Element* getEdge(unsigned i) const;
```

Comment:
The implementation of `getEdge` is the same as in `EdgeReturn.h`, `LinearEdgeReturn::getEdge`.
I don't yet see the big picture, but maybe like in the `TemplateElement` class this should return
`LinearEdgeReturn::EdgeReturn::getEdge(this, i)`?